### ISSUE TITLE
GSE103892  - Sathyamurthy2018MouseSpinal

### ISSUE THREAD

**Issue by ofanobilbao** [ebi-ait/hca-ebi-wrangler-central#950]
### Project short name:
Sathyamurthy2018MouseSpinal

### Primary Wrangler:
Ida
### Link to Ingest
* https://contribute.data.humancellatlas.org/projects/detail?uuid=cdc2d270-6c99-4142-8883-9bd95c041d05&tab=project

### Associated files

* Google Drive: [folder](https://drive.google.com/drive/folders/1baat-PLuSBtpkJvPAvctKeo3UphEJgbL)

### Published study links

* Paper: [Massively Parallel Single Nucleus Transcriptional Profiling Defines Spinal Cord Neurons and Their Activity during Behavior](https://doi.org/10.1016/j.celrep.2018.02.003) 

* Accessioned data: [GSE103892](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103892)

### Key Events

- [x] Convert published metadata to HCA spreadsheet
- [x] Manually curate dataset to meet HCA metadata standard
- [x] Collect any matrix and cell-type annotation files
- [x] Upload sheet to validate metadata
- [x] Check linking using ingest graph validator
- [x] Transfer raw files to ingest to validate data files
- [x] Ask the Secondary Wrangler for an end-to-end review of the project. Ask the Expertise Wrangler to review specific tabs if needed
- [ ] Submit dataset to Production 
- [ ] Complete the Export SOP
- [ ] Convert project data to SCEA format following the SCEA conversion SOP if appropriate

**Comment by MightyAx** [comment]
May be available for secondary review today.
- [ ] @idazucchi to make available for secondary reviewing
- [ ] @Wkt8 to begin secondary reviewing

**Comment by idazucchi** [comment]
Graph valid! This is ready for secondary review

**Comment by Wkt8** [comment]
Could not find the excel spreadsheet in the google drive folder so i downloaded it from the ingest endpoint

**Comment by Wkt8** [comment]
Notes:
Need to add yourself as a contributor! 
The numbers of mice are a bit confusing as they don't match up to the number of mice described in the methods. E.g. 10 formalin-treatment mice in the GEO accession but only 5 are mentioned in the paper in the Methods section. 
- I found this at the bottom of one of the GEO accessions (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM2785683) *For all formalin samples, a second round of sequencing was performed, producing ten sequencing reactions for the five biological replicates. 
- I think what this means is that the five extra formalin-treatment mice are not additional donors, but are biological replicates (sequenced again from the same cell suspensions). 
- This will, however, make the linking more difficult as you don't know which new set of sequence file was linked to which cell suspension. You might have to email about this. 

I couldn't see this information from the protocol stored anywhere (12mL of density buffer (1M sucrose, 10mM HEPES pH 8.0, 3mM Mg-Acetate, 1mM DTT) was then added carefully below the nuclei layer and the tube was centrifuged at 3,200xg for 20 minutes. The supernatant was then rapidly poured off and the nuclei on the walls of the tube were collected with 1mL of PBS with 0.01% BSA and spun at 3,200xg for 10 minutes. Nuclei were then resuspended in PBS with 0.01% BSA.)

Everything else looks good!!
